demo/monolayer_growth/src/behaviours.h

```diff
@@ -0,0 +1,115 @@
+// -----------------------------------------------------------------------------
+//
+// Copyright (C) 2021 CERN & University of Surrey for the benefit of the
+// BioDynaMo collaboration. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// See the LICENSE file distributed with this work for details.
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// -----------------------------------------------------------------------------
+
+#ifndef BEHAVIOURS_H_
+#define BEHAVIOURS_H_
+
+#include "biodynamo.h"
+#include "core/behavior/behavior.h"
+#include "cycling_cell.h"
+#include "sim_param.h"
+
+namespace bdm {
+
+// Define growth behaviour
+struct Growth : public Behavior {
+  BDM_BEHAVIOR_HEADER(Growth, Behavior, 1);
+
+  Growth() { AlwaysCopyToNew(); }
+  virtual ~Growth() {}
+
+  void Run(Agent* agent) override {
+    if (auto* cell = dynamic_cast<CyclingCell*>(agent)) {
+      auto* random = Simulation::GetActive()->GetRandom();
+      real_t ran = random->Uniform(0, 1) * 1.0;
+      auto* sparam =
+          Simulation::GetActive()
+              ->GetParam()
+              ->Get<SimParam>();  // get a pointer to an instance of SimParam
+
+      // Counter for Delta t at each stage
+      // Used for calculating probability of moving to next state.
+      cell->SetDelt(cell->GetDelt() + 0.1);
+
+      // If statements for checking what states we are in and if
+      // a cell moves to the next state based on cumulative probability.
+      if (cell->GetCycle() == 1) {
+        real_t p1 = (cell->GetDelt() / 7) * cell->GetDelt();
+        if (p1 > ran) {
+          // Changing cells state number or "cycle" postiion.
+          cell->SetCycle(2);
```

Comment:
Here the calls would read `cell->GetCycle == CellState:kG0` and `cell->SetCycle(CellState::kG1)` or whatever `1` and `2` refers to if we adapt the changes suggested in `cycling_cell.h`.